quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -26,6 +27,17 @@ public class CachingOptions {
     private static final String CACHE_METRICS_PREFIX = "cache-metrics";
     public static final String CACHE_METRICS_HISTOGRAMS_ENABLED_PROPERTY = CACHE_METRICS_PREFIX + "-histograms-enabled";
 
+    public static final String[] MAX_COUNT_CACHES = new String[]{"authorization", "keys", "realms", "users", "actionTokens",
+          "authenticationSessions", "clientSessions", "loginFailures", "offlineSessions", "offlineClientSessions", "sessions"};
```

Comment:
`actionToken`, `authenticationSessions`, and `loginFailures` data are not stored in the database. Those cannot be bounded.